src/main/reactjs/src/configs/i18n.ts

```diff
@@ -9,6 +9,7 @@ import LanguageDetector from 'i18next-browser-languagedetector';
 import transFI from 'public/i18n/fi-FI.json';
 import transSV from 'public/i18n/sv-SE.json';
 import transEN from 'public/i18n/en-GB.json';
+import { Language } from '../interfaces/translator';
```

Comment:
importeissa ollaan pyritty käyttämään absoluuttisia polkuja => `interfaces/translator`

Reply:
Jes, ja löysin asetuksen jolla ide tekee importin näin. Eikö tähän ole lint sääntöä olemassa? Yritin etsiä, en löytänyt.

Reply:
```
    "no-restricted-imports": [
      "error",
      {
        "patterns": [".*"]
      }
    ]
```

Tällä pitäisi onnistua. Lisään tämänkin taskin 156 yhteydessä.